src/simplewallet/simplewallet.cpp

```diff
@@ -164,7 +167,7 @@ static std::string get_human_readable_timespan(uint64_t seconds);
 
 namespace
 {
-  const std::array<const char* const, 5> allowed_priority_strings = {{"default", "unimportant", "normal", "elevated", "priority"}};
+  const std::array<std::string_view, 5> allowed_priority_strings = tools::fee_priority_utilities::fee_priority_strings;
```

Comment:
Missed this last time. This should be marked `constexpr` as well.

Reply:
Thank you, done. Squashed and pushed.